app/navigation/Navigator.scala

```diff
@@ -198,6 +196,11 @@ class Navigator @Inject()(appConfig: FrontendAppConfig)
     }
   }
 
+  private def skipPayPeriodsIfFastJourney: UserAnswers => Call = { userAnswers =>
```

Comment:
minor: but the method impl is nothing to do with the fast track journey though , you can name as `employeeStartDateRoutes`